webpack.config.ts

```diff
@@ -9,6 +9,7 @@ import {ESBuildPlugin} from 'esbuild-loader';
 import CopyWebpackPlugin from 'copy-webpack-plugin';
 import MiniCssExtractPlugin from 'mini-css-extract-plugin';
 import webpack, {Configuration} from 'webpack';
+import * as markdown from 'markdown-wasm/dist/markdown.node';
```

Comment:
Can you use `import {parse as parseMarkdown}` instead? Also move the import higher in the list to align the `from` keyword